src/components/plots/scatterPlot.ts

```diff
@@ -87,6 +81,26 @@ export module Plots {
 
       return Utils.Methods.intersectsBBox(xRange, yRange, translatedBbox);
     }
+
+    protected _generatePropertyToProjectors(): AttributeToProjector {
+      var propertyToProjectors = super._generatePropertyToProjectors();
+
+      var xProjector = propertyToProjectors["x"];
+      var yProjector = propertyToProjectors["y"];
```

Comment:
Hmm, if we're just going to use the strings `"x"` and `"y"` here, should we remove the `static` keys from the superclass?

Alternatively, the `static` keys should be public, and we should use them here.